### ISSUE TITLE
RSS feed broken (and I believe other things may be also)

### ISSUE THREAD

**Issue by srynot4sale** [kenanb/kenanb-blog#1]
The RSS feed detector I use ( https://github.com/shevabam/get-rss-feed-url-extension ), as well as my feed reader, struggle with the URLs presented.

They appear as https://blog.kenanb.com/blog.kenanb.com/feed.xml, https://blog.kenanb.com/blog.kenanb.com/postXXX, etc

I believe it's due to a misconfiguration in Jekyll.

```
<link rel="canonical" href="blog.kenanb.com/" />
...
{"@context":"https://schema.org","@type":"WebSite","author":{"@type":"Person","name":"Kenan Bölükbaşı"},"description":"A blog about programming and graphics.","headline":"Kenan Bölükbaşı - Blog","name":"Kenan Bölükbaşı - Blog","url":"blog.kenanb.com/"}</script>
...
<link type="application/atom+xml" rel="alternate" href="blog.kenanb.com/feed.xml" title="Kenan Bölükbaşı - Blog" /></head>
```

Note the URLs don't begin with `https://` or `/` or `//` which I'd expect them to. So instead they are treated relative to the current page, e.g https://blog.kenanb.com/blog.kenanb.com/XXX

So sounds like you need to add https:// to the setting in Jekyll that configures the base url of the site.

**Comment by kenanb** [comment]
Thanks a lot for pointing this out @srynot4sale !

Yes, I was indeed omitting the protocol in the `url` config property. I checked github pages documentation here to confirm what it should look like: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll

And it does mention the protocol.
> url: https://my-site.github.io  # the base hostname and protocol for your site, e.g. http://example.com

So I just revised that and checked the page with the Chrome extension you linked. It now seems to point at a valid URL: https://blog.kenanb.com/feed.xml

(I had to clean up my browser cache for the extension to find the correct address.)

Thanks for reporting! I am closing this as completed.
